services/motion/builtin/state/state.go

```diff
@@ -430,6 +430,16 @@ func (s *State) PlanHistory(req motion.PlanHistoryReq) ([]motion.PlanWithStatus,
 	return history, nil
 }
 
+// IsActive returns a bool indicating if we have active execution for any component.
+func (s *State) IsActive() bool {
```

Comment:
Where is this used?

Reply:
Is this used somewhere @nfranczak ?

Reply:
nope, it has been removed.
I added this when I was figuring out why the service was persistently reconfiguring.